src/montgomery.rs

```diff
@@ -155,6 +155,12 @@ impl Identity for ProjectivePoint {
     }
 }
 
+impl Default for ProjectivePoint {
+    fn default() -> ProjectivePoint {
+        ProjectivePoint::identity()
+    }
+}
+
```

Comment:
Should we add this? We don't expose the `ProjectivePoint` externally, and we don't use the `Default` impl internally, so this function is dead code.